panels/site_sections/panel_app_management.py

```diff
@@ -72,17 +67,99 @@ def associate(self, session, message='', **params):
             if not app.event_id:
                 message = 'You must select an event'
             else:
-                for attendee in app.matching_attendees:
+                for attendee in app.matched_attendees:
                     if not session.query(AssignedPanelist).filter_by(event_id=app.event_id, attendee_id=attendee.id).first():
                         app.event.assigned_panelists.append(AssignedPanelist(attendee=attendee))
                 raise HTTPRedirect('index?message={}{}{}', app.name, ' was associated with ', app.event.name)
 
         return {
             'app': app,
             'message': message,
-            'panels': [e for e in session.query(Event).filter(Event.location.in_(c.PANEL_ROOMS)).order_by('name').all()]
+            'panels': session.query(Event).filter(Event.location.in_(c.PANEL_ROOMS)).order_by('name')
         }
 
+    def badges(self, session):
```

Comment:
This page replaces the Bad Old Way of automatically creating badges and groups for panelists.  Instead, admins will see a list of all possible matches and must manually click a button for each to indicate that the panelist should be associated with a particular attendee OR that a new badge should be created for them.